config/schema_descriptions.yml

```diff
@@ -0,0 +1,72 @@
+reporting_patient_states:
+  description: "Monthly summary of a patient's information and health indicators. This table has one row per patient, per month, from the month of the patient's registration."
+  columns:
+    patient_id: "ID of the patient"
+    recorded_at: "Time (in UTC) at which the patient was registered"
+    status: "active, dead, migrated, etc"
+    gender: "Gender of the patient"
+    age: "Age of the patient as entered by a nurse"
+    age_updated_at: "Time (in UTC) at which the age was last updated"
+    date_of_birth: "Date of birth of the patient"
+    current_age: "Computed age as of today, given age, age-updated-at and date-of-birth. This will have the same value for a given patient across all rows."
+    month_date: "Date at the beginning of the month for which this row has the latest information of a patient"
```

Comment:
I wonder if a simpler description would be more effective for and epi audience. Eg. "The reporting month of this row". We can then utilize "the reporting month" as a consistent reference down below, eg. for the treatment outcomes and stuff.

Reply:
I've updated the definition here. But I like "this month" in other descriptions because it feels simpler, easier to relate to. Wdyt?

Reply:
I think that sounds great 👍  PS the updated description for `month_date` is :chef_kiss: